uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/rules/RetryTestRule.java

```diff
@@ -36,6 +36,7 @@
 public class RetryTestRule implements TestRule {
 
     private final static String TAG = RetryTestRule.class.getSimpleName();
+    private final static int MINIMUM_NUMBER_OF_ATTEMPTS = 3;
```

Comment:
3 seems quite a bit tbh. This is going to increase the bill. Are we okay with that?

Reply:
If a test is consistently failing with 3 retries then we MUST actively look at test reports and fix it right away otherwise we're just wasting money running bad test cases. Looking at daily test reports becomes very important with this.